### PR TITLE
fix: Update invoice management How-To guide

### PR DESCRIPTION
**Pull request by colder-is-better**
We add a section showing how to turn automatic payments on or off.

### CODE REVIEW

File: docs/howto/account-billing/manage-invoices.md [hunk 1]
```diff
@@ -31,3 +31,13 @@ There is also the _Pay this invoice_ option.
 Choose it for an outstanding invoice, and you will get a pop-up with a detailed rundown of the dues, together with available payment methods.
 
 ![Payment options](assets/mgmnt-inv/shot-04.png)
+
+## Activate automatic payments
+
+You may activate automatic payments for pending invoices.
+From the left-hand side vertical pane, click on *Settings* and then on *Manage Invoice Settings*.
```
Comment (fghaas) [inline docs/howto/account-billing/manage-invoices.md:38]: I'd add a newline between the first and second sentence here.
```suggestion
You may activate automatic payments for pending invoices.

From the left-hand side vertical pane, click on *Settings* and then on *Manage Invoice Settings*.
```
What do you think?

**Review by fghaas — COMMENTED** [review]
Looks good; I have one tiny nit.

File: docs/howto/account-billing/manage-invoices.md (hunk 1 above)
Comment (colder-is-better) [reply docs/howto/account-billing/manage-invoices.md:38]: It's better, yes.

**Review by fghaas — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**